client/scss/components/_body_content.scss

```diff
@@ -94,6 +94,10 @@ $center-width-xlarge: (6/6) * 100%;
       clear: right;
       width: #{$narrow-width};
       margin-right: calc((#{$narrow-width} + #{map-get($gutter-width, 'large')} ) * -1);
+
+      .author + .article &:first-child {
```

Comment:
Would adding a class to this and target that be safer?
e.g. if the first child is the paragraph - do we still want to do this?